android/main/java/com/couchbase/lite/CouchbaseLite.java

```diff
@@ -24,49 +24,98 @@
 import com.couchbase.lite.internal.CouchbaseLiteInternal;
 
 
+/**
+ * One of the methods from this utility class MUST be called prior to any use of CouchbaseLite.
+ * If you intend to use Vector Search, you must use one of the two methods that specify a
+ * extDir for the vector search library.  To use the default search path for the vector
+ * search library specify VS_USE_DEFAULT as the extDir.
+ */
 public final class CouchbaseLite {
     // Utility class
-    private CouchbaseLite() {}
+    private CouchbaseLite() { }
+
+    /**
+     * Use VS_USE_DEFAULT to specify that Couchbase Lite should look for extension libraries in the default location
+     */
+    public static final File DEFAULT_EXT_DIR = new File("*");
 
     /**
      * Initialize CouchbaseLite library. This method MUST be called before using CouchbaseLite.
      *
      * @param ctxt the ApplicationContext.
-     * @throws IllegalStateException on initialization failure
      */
     public static void init(@NonNull Context ctxt) { init(ctxt, BuildConfig.CBL_DEBUG); }
 
     /**
-     * Initialize CouchbaseLite library. This method MUST be called before using CouchbaseLite.
+     * Initialize CouchbaseLite library. Call this method before using CouchbaseLite.
      *
+     * @param ctxt  the ApplicationContext.
      * @param debug true to enable debugging
-     * @throws IllegalStateException on initialization failure
      */
     public static void init(@NonNull Context ctxt, boolean debug) {
-        init(ctxt, debug, ctxt.getFilesDir(), new File(ctxt.getFilesDir(), CouchbaseLiteInternal.SCRATCH_DIR_NAME));
+        final File filesDir = ctxt.getFilesDir();
+        init(ctxt, DEFAULT_EXT_DIR, debug, filesDir, new File(filesDir, CouchbaseLiteInternal.SCRATCH_DIR_NAME));
```

Comment:
Doesn't this bring back the issue yesterday of "there is no way to tell if the developer meant to use the extension directory or not"?

Reply:
no....   client code can't explicitly specify default.   I need to check that.